markdown/4.x/en/ff-suggest.md

```diff
@@ -199,3 +199,18 @@ See the following example:
     });
 </script>
 ```
+
+## On-Click redirect
+The expected behavior after clicking on `ff-suggest-item` element is a redirection to the clicked record's page.
+`ff-suggest` support this functionality if only `deeplink` field is available in clicked item attributes data.
+The `deeplink` field is populated using the data from data feed export column with a field role with the same name.
+
+**Note**: `ff-suggest` is redirecting both absolute and relative URLs.
+
+If no `deeplink` is provided for anitem which is of `productName` type then `ff-suggest` will try to obtain its value by emitting additional request.
+Depending on used FACT-Finder version it would be a search - for versions lower than `7.3`, or record API call - for newer versions.
+Then`ff-suggest` will try to redirect user once more, this time using the data received from the response to the mentioned request.
```

Comment:
Parentheses might be a little clearer about their reference than dashes.
```suggestion
Depending on the used FACT-Finder version it will be a search request (versions lower than `7.3`), or a records API call (versions `7.3` and up).
Then `ff-suggest` will try to redirect the user once more, this time using the data received from the response to the mentioned request.
```